contracts/bonding/BondingManager.sol

```diff
@@ -551,6 +548,8 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
 
                 TranscoderSlashed(_transcoder, address(0), penalty, 0);
             }
+        } else {
+            TranscoderSlashed(_transcoder, _finder, 0, 0);
```

Comment:
Still fire an event even if the user being slashed no longer has a bonded amount (the edge case in which the slashing transaction was not processed until the transcoder already unbonded, waited through the unbonding period and withdrew its stake)